tests/external_pyzeebe/test_client.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import asyncio
+import sys, types
+import _dummy_client as dummy_client
+
+from newrelic.api.background_task import background_task
+
+# Force import system to use dummy ZeebeClient
+pyzeebe_mod = types.ModuleType("pyzeebe")
+pyzeebe_client_pkg = types.ModuleType("pyzeebe.client")
+
+pyzeebe_mod.client = pyzeebe_client_pkg
+pyzeebe_client_pkg.client = dummy_client
+
+sys.modules["pyzeebe"] = pyzeebe_mod
+sys.modules["pyzeebe.client"] = pyzeebe_client_pkg
+sys.modules["pyzeebe.client.client"] = dummy_client
+
+from pyzeebe.client.client import ZeebeClient
+from testing_support.validators.validate_transaction_metrics import validate_transaction_metrics
+
+@validate_transaction_metrics(
```

Comment:
We will also want to test for the presence of the agent attributes added as well. We have validators for that, here are a few examples: https://github.com/newrelic/newrelic-python-agent/blob/main/tests/messagebroker_kombu/test_consumer.py#L105, https://github.com/newrelic/newrelic-python-agent/blob/main/tests/mlmodel_openai/test_chat_completion.py#L138

Reply:
fixed in latest commit